### DISCUSSION TITLE
How do full searches work?

### DISCUSSION THREAD

**Discussion by bavis-m** [TriliumNext/Trilium#1999] [Q&A] [answered]
Hitting the Search button at the top (or using Ctrl-S, although for some reason that shortcut isn't actually working for me right now) appears to bring up an actual note which is used to do full search (a feature I find I want to use a lot). However, the first time I used full search, the note appeared as a child note of a random note several levels deep in the Trilium sample notes that shipped with the initial application install (I'm using a server-based install and the electron Windows desktop client). I've since deleted the default Trilium sample notes, but whenever I hit the search button, it restores (from backup/history presumably) the branch (and *only* the notes in that branch) that the search note used to appear as a child of, and adds the search not to this exact same location again. Ctrl-S now no longer brings up full search (although I don't know if it's related to me deleting the sample notes, or if it only starting happening coincidentally lat the same time). How can I change where the search note appears? Why does full search need to be a full first-class note anyways?

**Comment by zadam** [comment, marked as answer]
Hi, `CTRL-S` now puts focus on "quick search". But you should be able to change this in settings.

"Full search" is a type of note. The reason for this was that you might want to save the search to use next time, so you would have 2 modes - "saved in a note" and "not represented in a note". Implementing only the first was sort of "cost/time saving" measure. But it's pretty unpopular so I will probably implement also search which does not have to be saved in a note.

The reason that the "Journal" structure gets recreated is that today's note is the default location for these saved notes. You can assign a label `#searchHome` to new note e.g. "Search history" and then all search notes will get saved there.

**Reply by bavis-m** [reply]
Ah ok thanks, not sure how I mixed up Ctrl-S and Ctrl-Shift-S. And the label #searchHome was what I needed, thanks!
